docs/using/installing/index.rst

```diff
@@ -506,6 +507,34 @@ For a ``loopback`` device, change the ``dataset`` clause to:
       "backend": "loopback"
       "root_path": "/var/lib/flocker/loopback"
 
+OpenStack Block Device Backend Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: yaml
+
+   dataset:
+       backend: "openstack"
+       region: "LON"
+       auth_plugin: "rackspace"
+       username: "joe.bloggs.rackspace"
+       api_key: "aaa-bbb-ccc-ddd"
+       auth_url: "https://identity.api.rackspacecloud.com/v2.0"
+
+
+Amazon AWS / EBS Block Device Backend Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: yaml
+
+   dataset:
+       backend: "aws"
+       region: "us-west-1"
+       zone: "us-west-1a"
+       access_key_id: "foo"
+       secret_access_key: "bar"
```

Comment:
Any way we can test this?  Can it be some kind of include of data that gets used by that smoke test I mentioned elsewhere?  Or ... doctest ...